### ISSUE TITLE
symbols like "&" in seed file breaks booting the AMI

### ISSUE THREAD

**Issue by baversjo** [remind101/empire_ami#15]
Placing special symbols like an `&` in the seed file prevents ansible from loading the seed file with the following message in ansible.log:

```
[Tue Jan  5 22:50:22 UTC 2016] Starting ansible
== Loading /etc/empire/seed
/proc/self/fd/9: 18: /etc/empire/seed: IGwvGqBx^: not found
```

Maybe the seed file should be generated with quotes around variable values, as suggested here if the value string contains special characters in a YAML file:
http://stackoverflow.com/questions/19109912/do-i-need-quotes-for-strings-in-yaml

**Comment by phobologic** [comment]
Hmm- so I took a look at this, and this is just an issue with us importing the seed as shell environment variables (https://github.com/remind101/empire_ami/blob/master/files/ansible.conf#L22).  An `&` on a line indicates the end of a command, so this is problematic.

I've toyed with it a bit, and I think we could do something like this instead:

```
for line in $(cat seed)
do
    IFS=\= read -r envvar value <<< "$line"
    export $envvar=$value
done
```

That would safely handle this use case, though I'm trying to figure out if there are any edge cases that might pop up.
